src/openrct2/ride/Ride.cpp

```diff
@@ -853,6 +861,11 @@ void ride_get_status(sint32 rideIndex, rct_string_id *formatSecondary, sint32 *a
 {
     Ride *ride = get_ride(rideIndex);
 
+    if (ride == nullptr)
```

Comment:
This may cause issues as formatSecondary and argument have not been set to anything. These values will be passed into the the window drawing code as uninitialised memory.